pubsub/streaming-analytics/README.md

```diff
@@ -0,0 +1,211 @@
+# Stream Cloud Pub/Sub with Cloud Dataflow
```

Comment:
Any reason you're not using README-gen and shipping this as restructured text / generated documentation?

Reply:
I avoided it because this tutorial does not a standard README. Much of it matches the Java tutorial's README.